tests/rest/client/v2_alpha/test_auth.py

```diff
@@ -59,24 +64,18 @@ def prepare(self, reactor, clock, hs):
         auth_handler = hs.get_auth_handler()
         auth_handler.checkers[LoginType.RECAPTCHA] = self.recaptcha_checker
 
-    @unittest.INFO
-    def test_fallback_captcha(self):
-
-        request, channel = self.make_request(
-            "POST",
-            "register",
-            {"username": "user", "type": "m.login.password", "password": "bar"},
-        )
+    def register(self, expected_response, body):
+        """Make a register request."""
```

Comment:
Could maybe do with documenting what this returns? Maybe even add some type annotations if you're feeling keen?

Reply:
Good call!